Tests/Resources/app/AppKernel.php

```diff
@@ -20,8 +20,7 @@ public function configure()
 
         if ('phpcr' === $this->environment) {
             $this->requireBundleSets(array(
-                'phpcr_odm',
-                'sonata_admin_phpcr',
```

Comment:
Btw: as wee need the set in sonata-admin-integration-bundle i would keep it in testing component, or do you think "it is just one usage"?